src/Native/pkg/runtime.native.System.Security.Cryptography/dir.props

```diff
@@ -0,0 +1,7 @@
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)\.., dir.props))\dir.props" />
+
+  <PropertyGroup>
```

Comment:
I didn't realize all of them were using this. In that case I like the common file.